cmd/conduit/root/connectors/describe.go

```diff
@@ -0,0 +1,159 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connectors
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/conduitio/conduit/cmd/conduit/api"
+	"github.com/conduitio/conduit/cmd/conduit/cecdysis"
+	"github.com/conduitio/conduit/cmd/conduit/internal"
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+	apiv1 "github.com/conduitio/conduit/proto/api/v1"
+	"github.com/conduitio/ecdysis"
+)
+
+var (
+	_ cecdysis.CommandWithExecuteWithClient = (*DescribeCommand)(nil)
+	_ ecdysis.CommandWithAliases            = (*DescribeCommand)(nil)
+	_ ecdysis.CommandWithDocs               = (*DescribeCommand)(nil)
+	_ ecdysis.CommandWithArgs               = (*DescribeCommand)(nil)
+)
+
+type DescribeArgs struct {
+	ConnectorID string
+}
+
+type DescribeCommand struct {
+	args DescribeArgs
+}
+
+func (c *DescribeCommand) Usage() string { return "describe" }
+
+func (c *DescribeCommand) Docs() ecdysis.Docs {
+	return ecdysis.Docs{
+		Short: "Describe an existing connector",
+		Long: `This command requires Conduit to be already running since it will describe a connector registered
+by Conduit. You can list existing connectors with the 'conduit connectors ls' command.`,
```

Comment:
I'd use `list` in the doc string, since that's the full command name, `ls` is only an alias.
```suggestion
by Conduit. You can list existing connectors with the 'conduit connectors list' command.`,
```